.github/workflows/test.yml

```diff
@@ -243,8 +243,10 @@ jobs:
         # in cibuildwheel/resources/build-platforms.toml.
       env:
         CIBW_PLATFORM: pyodide
-        CIBW_BUILD: "cp312*"
-        CIBW_PYODIDE_VERSION: "0.27.6"
+        CIBW_BUILD: "cp313*"
```

Comment:
Why not just leave it as is?

Reply:
I was gonna say, we could just remove this whole build step now? I think everything is covered in integration tests anyway now.